java/org/contikios/cooja/Cooja.java

```diff
@@ -863,6 +863,49 @@ public Plugin getPlugin(String classname) {
     return null;
   }
 
+  /**
+   * Returns a started plugin that is assignable by the given class, if any.
+   *
+   * @param type a class specifying the type of plugin
+   * @return Plugin instance
+   */
+  public <T extends Plugin> T getPlugin(Class<T> type) {
```

Comment:
The parameter should be called clazz rather than type.

Reply:
Renamed it `pluginClass`.